docs/changelog.rst

```diff
@@ -4,10 +4,15 @@ Changelog
 
 .. seealso:: :ref:`howto_upgrade`
 
+.. |development_version| replace:: 1.1.2alpha1
+
 Development version (|development_version|)
 ===========================================
 
-.. |development_version| replace:: 1.1.2alpha1
+Deprecated
+----------
+
+* A reminder to not install the theme directly from Git, we will remove support for this in 3.0
```

Comment:
Currently, this section looks a bit empty, so it's nice to have a single item: https://sphinx-rtd-theme.readthedocs.io/en/latest/changelog.html

Reply:
Maybe the text should be phrased a bit more "clearly"

```suggestion
* Reminder: We will remove support for installing this theme directly from its git repository. Instead, please use releases from PyPi.
```